src/main/java/ua/edu/knightandwarrior/model/units/IWarrior.java

```diff
@@ -0,0 +1,16 @@
+package ua.edu.knightandwarrior.model.units;
+
+public interface IWarrior {
+    default boolean isAlive() {
+        return getHealth()>0;
```

Comment:
Can we have the zero as constant, please

Reply:
Thanks, I fixed it